packages/mattermost-redux/src/client/client4.ts

```diff
@@ -1992,11 +1992,11 @@ export default class Client4 {
         );
     };
 
-    getFlaggedPosts = (userId: string, channelId = '', teamId = '', page = 0, perPage = PER_PAGE_DEFAULT) => {
-        this.trackEvent('api', 'api_posts_get_flagged', {team_id: teamId});
+    getFlaggedPosts = (userId: string, channelId = '', page = 0, perPage = PER_PAGE_DEFAULT) => {
```

Comment:
0/5 but do we want to leave the Client4 route as-is? I'm wondering if we might run into a situation in the future where we want one part of the API to show flagged posts for all teams and another part where we only show for one team.

Reply:
@esethna :point_up: ?

Reply:
@larkox @hmhealey I could imagine a future where we offer a filter mechanism in the saved posts view to see flagged posts from only certain teams